SYCL/InvokeSimd/invoke_simd_smoke.cpp

```diff
@@ -97,7 +72,8 @@ template <bool use_func_directly> bool test() {
   constexpr unsigned Size = 1024;
   constexpr unsigned GroupSize = 4 * VL;
 
-  queue q(ESIMDSelector{}, createExceptionHandler());
+  auto GPUSelector = gpu_selector{};
```

Comment:
why these changes to the existing tests?

Reply:
This way it is possible to avoid using ESIMDSelector and createExceptionHandler, that are described in SYCL/ESIMD/esimd_test_utils.hpp.

esimd_test_utils.hpp seems to be a bit overkill for these tests, as none of it is expected to run on host device, only on gpu. So I decided it would be reasonable to hardcode gpu selector.